### PR TITLE
Added a Quote Generator project

### PR DESCRIPTION
**Pull request by skyHunter-18**
Added a new project called Quote Generator made using HTML, CSS and JS which uses a Quotes API to fetch quotes.

PS: Please add the Hacktoberfest label.

### CODE REVIEW

**Comment by skyHunter-18** [comment]
Hi @iamwatchdogs . Sorry I'm new to this so I have a doubt. As I see all the **eslint** errors are coming from E Commerce project. Since that is an existing file what should I do? Can you please help me out here. Thanks.

**Review by iamwatchdogs — APPROVED** [review]
Reviewed and Approved by @iamwatchdogs.

File: quote-generator/index.md
Comment (iamwatchdogs) [inline quote-generator/index.md:1]: Please rename the `index.md` to `README.md` as it might be an issue while deploying your work to GitHub Pages.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]

File: quote-generator/index.md
Comment (skyHunter-18) [reply quote-generator/index.md:1]: Renamed the file. Please check.

**Review by iamwatchdogs — APPROVED** [review]
Requested Changes have been made. Reviewed and Approved by @iamwatchdogs.

**Comment by iamwatchdogs** [comment]
Thank you for contributing, @skyHunter-18. Please check your contribution on [GitHub Pages](https://grow-with-open-source.github.io/Javascript-Projects/).

**Comment by skyHunter-18** [comment]
Thanks @iamwatchdogs . This was my first Open Source Contribution.

**Comment by iamwatchdogs** [comment]
Hey @skyHunter-18, good job on your first Open Source Contribution. I hope you had fun while contributing something from your side to our repos. 

We're sorry about the linter, as initially we configured it to do some strict linting checks and these configure are based on pure JavaScript. But the PR #6 project was built using jQuery and didn't pass the linting checks due to its non-pure-javascript syntax, that's the reason we have reconfigured the linter to perform not-so-strict linting.

Once again thank you for your contributions, hope you had fun participating in our hacktoberfest challenge. As a request from @Grow-with-Open-Source, can you create posts sharing your experience for each pull request that is merged on your socials _(link LinkedIn)_? One PR per post, so that you can get more impressions for your account and don't forget to tag us in.

Thank you for being an active contributor to our community.

**Pull request merged**